pnpm-lock.yaml

```diff
@@ -12374,20 +12130,20 @@ snapshots:
   jsdom@20.0.3:
     dependencies:
       abab: 2.0.6
-      acorn: 8.12.1
+      acorn: 8.14.0
       acorn-globals: 7.0.1
       cssom: 0.5.0
       cssstyle: 2.3.0
       data-urls: 3.0.2
       decimal.js: 10.4.3
       domexception: 4.0.0
       escodegen: 2.1.0
-      form-data: 4.0.0
+      form-data: 4.0.1
       html-encoding-sniffer: 3.0.0
       http-proxy-agent: 5.0.0
       https-proxy-agent: 5.0.1
       is-potential-custom-element-name: 1.0.1
-      nwsapi: 2.2.2
+      nwsapi: 2.2.13
```

Comment:
Ok, @bl-nero and I were able to figure out what happened with tests that are using `toHaveStyle`.

After I deduped packages, it bumped nwsapi used by JSDOM v20 to v2.2.13. This version of JSDOM is used by jest-environment-jsdom in test. In our Vite config we use JSDOM v25.

nwsapi has an issue which causes the element to always render with `:active` and `:hover` styles (https://github.com/dperini/nwsapi/issues/119). According to [a comment under an issue in JSDOM repo](https://github.com/jsdom/jsdom/issues/3607#issuecomment-2227704965), it was fixed in 2.2.8 and then broken again 2.2.10.

I saw that other people have taken to force the version of nwsapi to 2.2.9 (e.g., https://github.com/streamlit/streamlit/pull/9165#pullrequestreview-2391201773). Until https://github.com/dperini/nwsapi/issues/119 is fixed, this seems to be the best way forward.

I was able to utilize some extra syntax offered by pnpm (or maybe all package managers do that now?) where the override is going to be used only by jsdom@^20.0.0.